src/github.com/getlantern/flashlight/android/bindings/main.go

```diff
@@ -0,0 +1,56 @@
+// package flashlight provides minimal configuration for spawning a flashlight
```

Comment:
This package defines the exported methods that are going to be available to Android developers using the NDK.